ts/parts-more/PackedBubbleSeries.ts

```diff
@@ -1891,6 +1875,27 @@ addEvent(Chart as any, 'beforeRedraw', function (
     }
 });
 
+
+// disable simulation before print if enabled
+addEvent(Chart as any, 'beforePrint', function (
+    this: Highcharts.PackedBubbleChart
+): void {
+    this.graphLayoutsLookup.forEach(function (layout): void {
+        layout.updateSimulation(true, false);
+    });
+    this.redraw();
+});
```

Comment:
Shouldn't this be defined in Layout? Does it fix networkgrap issue when packed bubble series is not included?

Reply:
I will move it to Layout, then Networkgraph print will be fixed as well.